pack/scw.json

```diff
@@ -0,0 +1,380 @@
+		"set_code": "scw",
+		"set_position": 11,
+		"text": "<b>Alter-Ego Action:</b> Exhaust Agatha Harkness → look at the top 3 cards of your deck. Add 1 of those to your hand and place the rest on the bottom of your deck in any order.",
+		"traits": "Persona.",
+		"type_code": "support"
+	},
+	{
+		"card_set_code": "scw",
+		"code": "15008",
+		"cost": 1,
+		"deck_limit": 3,
+		"faction_code": "hero",
+		"illustrator": "Ramon Bachs",
+		"name": "Magic Shield",
+		"pack_code": "scw",
+		"position": 8,
+		"quantity": 3,
+		"resource_physical": 1,
+		"set_code": "scw",
+		"set_position": 12,
+		"text": "<b>Hero Interrupt:</b> When a friendly character would take any amount of damage, discard Magic Shield → prevent 3 of that damage.",
+		"traits": "Spell.",
+		"type_code": "upgrade"
+	},
+	{
+		"card_set_code": "scw",
+		"code": "15009",
+		"cost": 2,
+		"deck_limit": 1,
+		"faction_code": "hero",
+		"illustrator": "Miguel Sepulveda",
+		"name": "Scarlet Witch's Crest",
+		"pack_code": "scw",
+		"position": 9,
+		"quantity": 1,
+		"resource_energy": 1,
+		"set_code": "scw",
+		"set_position": 15,
+		"text": "<b>Interrupt:</b> When boost icons on an encounter card are counted, exhaust Scarlet Witch's Crest → increase or decrease the number of boost icons on that card by 1 for this count.",
+		"traits": "Item.",
+		"type_code": "upgrade"
+	},
+	{
+		"attack": 1,
+		"attack_cost": 1,
+		"code": "15010",
+		"cost": 4,
+		"deck_limit": 1,
+		"faction_code": "justice",
+		"flavor": "\"Hey! Geriatrics! Let's get a move on. The Avengers need us!\"",
+		"health": 4,
+		"is_unique": true,
+		"name": "Speed",
+		"pack_code": "scw",
+		"position": 10,
+		"quantity": 1,
+		"resource_physical": 1,
+		"subname": "Thomas Shepherd",
+		"text": "<b>Response</b>: After Speed thwarts, ready him. (Limit once per round.)",
+		"thwart": 2,
+		"thwart_cost": 1,
+		"traits": "Avenger.",
+		"type_code": "ally"
+	},
+	{
+		"attack": 1,
+		"attack_cost": 1,
+		"code": "15011",
+		"cost": 2,
+		"deck_limit": 1,
+		"faction_code": "justice",
+		"flavor": "\"I'm the child of a witch and an android. I'm their wish come true.\"",
+		"health": 3,
+		"illustrator": "Steve Ellis",
+		"is_unique": true,
+		"name": "Wiccan",
+		"pack_code": "scw",
+		"position": 11,
+		"quantity": 1,
+		"resource_mental": 1,
+		"subname": "William Kaplan",
+		"text": "<b>Response:</b> After Wiccan thwarts, discard the top card of the encounter deck. For each boost icon discarded this way, deal 1 damage to an enemy.",
+		"thwart": 1,
+		"thwart_cost": 1,
+		"traits": "Avenger.",
+		"type_code": "ally"
+	},
+	{
+		"code": "15012",
+		"cost": 3,
+		"deck_limit": 3,
+		"faction_code": "justice",
+		"name": "Crisis Averted",
+		"pack_code": "scw",
+		"position": 12,
+		"quantity": 3,
+		"resource_mental": 1,
+		"text": "<b>Hero Action</b> <i>(thwart)</i>: Remove 6 threat from the main scheme. If you paid for this card using a [mental] resource, this thwart ignores the crisis icon ([crisis]).",
```

Comment:
I'm not sure if the `[crisis]` icon is supported as I could not find any usage of it in the code.

Reply:
It does not look like it is currently supported, but _should_ be a pretty straight-forward update in the db code. See here for the icon replacements that the app currently handles:

https://github.com/zzorba/marvelsdb/blob/7a94978e2c7ab267443bd5c9e45cb38c95d95681/src/AppBundle/Services/CardsData.php#L34